packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```diff
@@ -13,6 +13,9 @@
     background-color: rgb(var(--venia-grey));
     border: solid 1px rgb(var(--venia-border));
     border-radius: 2px;
+    height: 100%;
+    object-fit: contain;
+    object-position: center;
```

Comment:
Small change from UX, vertically center all images

Reply:
That should work better. 👍